app/2.0/docs/devguide/style-shadow-dom.md

```diff
@@ -515,6 +515,64 @@ class myElement extends Polymer.Element(){
 }
 ```
 
+### Style directional text using the :dir() selector
+
+The `:dir()` CSS selector allows for styling text specific to its orientation 
+(right-to-left or left-to-right). See the [documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir) for more information on the :dir() 
+selector.
+
+For elements that extend `Polymer.Element`, add the `Polymer.DirMixin` mixin to use
+`:dir()` styling.
+
+`index.html` { .caption}
+```html
+<!doctype html>
+<html lang="en" dir="rtl">
+  <head>
+    ...
+    <link rel="import" href="using-dir-selector.html">
+  </head>
+  <body>
+    <using-dir-selector></using-dir-selector>
+  </body>
+</html>
+```
+
+`using-dir-selector.html` { .caption}
+```html
+<link rel="import"  href="../polymer/polymer-element.html">
+<link rel="import"  href="../polymer/lib/mixins/dir-mixin.html">
+...
+<template>
+  <style>
+    :host {
+      display: block;
+      color: blue;
+    }
+    :dir() {
```

Comment:
Should be `:dir(rtl)` or `:dir(ltr)`.

Could also be `:host(:dir(rtl))` ... I'd be inclined to use this, because it's more parallel to the previous ruleset.

`:dir(rtl)` matches anything in the shadow tree (if the current document is rtl, or the current element has `dir=rtl` on it). There's an implied wildcard selector 

`:host(:dir(rtl))` matches the host itself  (if the current document is rtl, or the current element has `dir=rtl` on it).